images/nginx/nginx.conf

```diff
@@ -13,6 +14,10 @@ http {
  server {
     listen 80;
     location / {
+       keepalive_requests 1000;
```

Comment:
if this is gunna be per user, and each command is one request, you may want to increase this. For example in the 2user file, theres more than 4000 commands per user

Reply:
This will be one per client, so we will probably re-establish connections every 1000 requests regardless of user. I guess we might as well just load it up super high.

Reply:
Yea, I'm not sure if there's any reason to not go balls to the wall with this since our workloads are known ahead of time. Can tune it based on performance though